app/views/content/train-to-be-a-teacher/if-your-application-is-unsuccessful.md

```diff
@@ -0,0 +1,79 @@
+---
+title: If your teacher training application is unsuccessful
+description: |-
+  Read about what to do if your teacher training application is unsuccessful. Find advice on why an application can be unsuccessful and how to get support.
+related_content:
+    Choosing the right teacher training course provider : "/blog/choosing-the-right-teacher-training-course-provider"
+    7 things to include in your teacher training application : "/blog/7-things-to-include-in-your-teacher-training-application"
+    11 top tips for interview success : "/blog/11-top-tips-for-interview-success"
+date: "2023-03-30"
+keywords:
+  - adviser
+  - advisor
+  - ITT
+  - Initial Teacher Training
+  - QTS
+  - Qualified Teacher Status
+  - Teacher Training Adviser
+  - application
+  - apply
+  - how to apply
+  - applying for teacher training
+promo_content:
+    - content/train-to-be-a-teacher/promos/adviser-promo-apply
+---
+
+Try not to worry - you can reapply for up to 4 courses at a time, as many times as you want. Many people are successful the second or third time around.
+
+It’s worth looking at the feedback the course provider has given to understand why your application was not accepted. This will help you to decide on your next steps.
+
+## Why was my application unsuccessful?
+
+There could be a number of reasons your application was unsuccessful:
+
+### The course is full
+
+Unfortunately, popular courses do fill up quickly. You can [search for other courses](https://www.gov.uk/find-postgraduate-teacher-training-courses) with a different provider if the one you applied for is now full.
+
+### You do not have the GCSEs at minimum grade 4 (C), or equivalent
```

Comment:
```suggestion
### You do not have the required GCSEs at minimum grade 4 (C), or equivalent
```
Does 'the GCSEs' sound a bit odd? Happy for this to be left as is if you think not!